main/ejml-core/src/org/ejml/data/DMatrixSparseTriplet.java

```diff
@@ -169,7 +171,7 @@ public void unsafe_set( int row, int col, double value ) {
         int index = nz_index(row, col);
         if (index < 0)
             addItem(row, col, value);
```

Comment:
not in the graph structure so you could check to see if it's zero here and not add it.